### PR TITLE
Use disinfect to check payload and remove sanitize

### PR DESCRIPTION
**Pull request by Jozzey**

### CODE REVIEW

File: server/routes/upload-document.route.js [hunk 1]
```diff
@@ -219,6 +219,9 @@ module.exports = [
     path: `${Paths.UPLOAD_DOCUMENT}`,
     handler: handlers.post,
     config: {
```
Comment (duncannixon) [inline server/routes/upload-document.route.js:221]: Add comment here and in the corresponding place in upload-photo to explain why this is required and that no text boxes should appear on this upload page

**Review by duncannixon — CHANGES_REQUESTED** [review]
Looks OK, just needs a bit of explanation to aid future maintenance, as per my PR comment.

File: server/routes/upload-document.route.js (hunk 1 above)
Comment (Jozzey) [reply server/routes/upload-document.route.js:221]: Added

**Review by duncannixon — APPROVED** [review]
Looks good

**Pull request merged**